operator/controllers/lifecycle/keptnappversion/reconcile_workloadsstate.go

```diff
@@ -43,7 +40,7 @@ func (r *KeptnAppVersionReconciler) reconcileWorkloads(ctx context.Context, appV
 		}
 
 		if !found {
-			r.EventSender.SendK8sEvent(phase, "Warning", appVersion, "NotFound", "workloadInstance not found", appVersion.GetVersion())
+			r.EventSender.SendK8sEvent(phase, "Warning", appVersion, apicommon.PhaseStateNotFound, "workloadInstance not found", appVersion.GetVersion())
```

Comment:
to stay consistent with other "not-found" event messages, we should use smth like:

```suggestion
			r.EventSender.SendK8sEvent(phase, "Warning", appVersion, apicommon.PhaseStateNotFound, fmt.Sprintf("could not find KeptnWorkloadInstance for KeptnWorkload: %s ", w.Name), appVersion.GetVersion())
```